demisto_sdk/commands/common/tools.py

```diff
@@ -654,6 +671,29 @@ def get_pack_names_from_files(file_paths, skip_file_types=None):
     return packs
 
 
+def filter_files_by_type(file_paths=None, skip_file_types=None) -> set:
+    """get set of files and return the set whiteout the types to skip
+
+    Args:
+    - file_paths (set): set of content files.
+    - skip_file_types List[str]: list of file types to skip.
+
+    Returns:
+    files (set): list of files whiteout the types to skip
+    """
+    if file_paths is None:
+        file_paths = set()
+    files = set()
+    for path in file_paths:
+        # renamed files are in a tuples - the second element is the new file name
+        if isinstance(path, tuple):
+            path = path[1]
```

Comment:
Why did we add this again?

Reply:
Because when you rename a file, the "get_modified_and_added_files" function returns it as a tuple

Reply:
👍